jacoco-maven-plugin/src/org/jacoco/maven/AbstractReportMojo.java

```diff
@@ -41,13 +41,15 @@
 	 * Name of the root node HTML report pages.
 	 * 
 	 * @parameter default-value="${project.name}"
+	 * @since 0.7.7
 	 */
 	String title;
 
 	/**
 	 * Footer text used in HTML report pages.
 	 * 
 	 * @parameter
+	 * @since 0.7.7
```

Comment:
@Godin Good idea. Haven't touched changelog yet for this PR (will conflict with master anyways).